data/presets/craft/atelier.json

```diff
@@ -0,0 +1,9 @@
+{
+  "icon": "maki-art-gallery",
+  "geometry": ["point", "area"],
+  "terms": ["art studio", "art workshop", "studio"],
+  "tags": {
+    "craft": "atelier"
+  },
+  "name": "Atelier"
```

Comment:
“Atelier” is a very obscure term, at least in American English (which is the default localization). If “art studio” corresponds reasonably well to the intended meaning of `craft=atelier`, then it would be a much better preset name. I can’t say for sure, though, because I myself am quite unfamiliar with “atelier”.